projects/sandbox/vizapp/vizapp/pages/data_summary/param_hist.py

```diff
@@ -0,0 +1,196 @@
+from pathlib import Path
+
+import h5py
+import ipywidgets as widgets
+import numpy as np
+from bokeh.layouts import row
+from bokeh.plotting import figure, show
+from IPython.display import clear_output
+from ipywidgets import Layout
+
+label = [
```

Comment:
Rather than defining the labels up here, you could do something in `__init__` like `self.labels = list(self.foreground.keys())`